tox.ini

```diff
@@ -46,6 +46,8 @@ commands = bash tools/pycbc_test_suite.sh
 [testenv:py-inference]
 deps =
     {[base]deps}
+    gsl
```

Comment:
@WuShichao This would try to install with pip, see at the top where conda_deps are given, not that uses a different variable.

Reply:
You probably should just add the items to tbe global conda_deps section anyway, otherwise bbhx will try to be installed for tests that don't have the prereqs. One alternative is to add the bbhx pip commands here rather than companion.txt.